apps/cookbook/src/app/showcase/card-showcase/card-showcase.component.ts

```diff
@@ -38,16 +38,10 @@ export class CardShowcaseComponent {
       defaultValue: 'elevated',
       type: ['elevated', 'flat', 'outlined'],
     },
-    {
-      name: 'flat',
-      description:
-        'Deprecation warning: use "variant" instead. \n(Optional) Remove elevation on the card. Default elevation = 2, flat = 0.',
-      defaultValue: 'false',
-      type: ['boolean'],
-    },
     {
       name: 'hasPadding',
-      description: '',
+      description:
+        'If set, the card will appear with a white space between the border and the content',
```

Comment:
Consider if this should be either "whitespace" or simply "spacing":
```suggestion
        'If set, the card will have spacing between the border and the content',
```